targeted/curl_command_generator.js

```diff
@@ -0,0 +1,29 @@
+//it will generate and copy curl commands based on the request
+//released under the Apache v2.0 licence.
+//You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+//auther:@haseebeqx
+
+function invokeWith(msg) {
+	string = "curl -i -s -k -X  '"+msg.getRequestHeader().getMethod().toString()+"'  \\\n";
+	header = msg.getRequestHeader().toString();
```

Comment:
Since the request line is not used, it's better to use ```getHeadersAsString()``` (instead of ```toString()```) which returns just the headers.